### ISSUE TITLE
Crash on the server start

### ISSUE THREAD

**Issue by Bulldog83** [PersonTheCat/CaveGenerator#2]
Minecraft 1.12.2
Forge 14.23.4.2705
CaveGenerator 0.2.1

https://gist.github.com/Bulldog83/7507300f18a1eba33a9963373a88f080

**Comment by PersonTheCat** [comment]
Damn. You're right. Literally, the game's main class is client side only. I'm not sure how this was able to work for me. Regardless, I'll have this fixed by the next update, which will probably come out tomorrow or so. Thanks for your help!

**Comment by Bulldog83** [comment]
Theoretically, your mod can only work on the server, without the need for its presence in the client, because it generates only the landscape, without adding any new blocks or any entities.

**Comment by PersonTheCat** [comment]
That's correct, although I haven't deliberately tried to look into that yet. This crash was actually just occurring when I tried to copy the presets from the jar into the preset folders. You just use a class loader to give you a stream of data from the input location and write that data to a file. The problem was that the class I used actually does not exist in the server side. I was just handling it how I did before with my other mod, but it was never an issue there because I only used the method on the client side. It wasn't really a problem with the "mod" mod, just IO stuff. 

Edit: and of course the reason I didn't reproduce the crash was because I didn't set up an entirely clean game directory. The files were already written from before.
